2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,22 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(arr) {
+newArr = [];
+
+let arr2 = [];
+arr2 = arr.filter(x => x.substring(0,1) === "A");
+newArr = arr2.filter(funky);
+return newArr;
+
+function funky(el){
```

Comment:
This function is never reached as the getSettlers function will stop executing on line 24 as it always returns a value